java/cdk8s-core/package.json

```diff
@@ -7,8 +7,9 @@
   "private": true,
   "scripts": {
     "install": "mvn install",
-    "import": "cdk8s import && rm -rf imports",
-    "build": "mvn clean compile && npm run import && npm run synth",
+    "cleanImports": "rm -rf src/main/java/imports && rm -rf src/main/resources/imports && rm -rf imports",
+    "import": "npm run cleanImports && cdk8s import k8s --language java",
```

Comment:
The `--language` in un-necessary because it already specified in the `cdk8s.yaml` file:

https://github.com/cdk8s-team/cdk8s-examples/blob/5bc2189c783ccd7bb7b4c8e6b5e993c6f38875ef/java/cdk8s-core/cdk8s.yaml#L2